### ISSUE TITLE
Only the "CollectCoverage" property is working

### ISSUE THREAD

**Issue by artoxian** [coverlet-coverage/coverlet#1414]
I'm following the guide [here](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-code-coverage?tabs=windows) to try and use coverlet to create a code coverage report.

In summary, I've run the following commands:

```
dotnet new xunit -n XUnit.Coverlet.MSBuild
cd XUnit.Coverlet.MSBuild
dotnet add package coverlet.msbuild
dotnet test /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura
```

The issue is with the final step, where the output is:

![image](https://user-images.githubusercontent.com/877863/203691998-4b33daca-578e-489f-b221-a3b820aa459a.png)

So the output is still the default "coverage.json" instead of the expected cobertura xml.

So I tested with some other properties, and they also don't work:

```
dotnet test /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura
```

And the output is exactly the same:

![image](https://user-images.githubusercontent.com/877863/203692421-bc2cfc73-1c9a-4c22-88d7-396194072539.png)

I've run the same thing on my home PC, and it is not having the same issues.

The only difference I can think of is that the PC with the issue is a Windows Server 2016 machine, and my home PC is a Windows 10.

I've tried a complete uninstall and reinstall of Visual Studio which didn't help.

Can anyone suggest a way I can get this to work?

**Comment by petli** [comment]
@artoxian  Do you have .NET 7 SDK installed on the machine where it's not working? See #1391 for a discussion of current problems with that SDK version, and links to workarounds.

**Comment by artoxian** [comment]
I do have .NET 7 SDK. That is annoying.

**Comment by john0879** [comment]
I'm having the same issue with .NET 6 and Visual Studio 17.4.

**Comment by john0879** [comment]
Tried this instead based on a comment in #1391 and it worked.

`dotnet test -p:CollectCoverage=true -e:CoverletOutputFormat=cobertura -e:CoverletOutput='../../TestResults/'`

**Comment by daveMueller** [comment]
Hi all, the issue was fixed by the microsoft guys with sdk 7.0.101 that was released last week https://github.com/microsoft/vstest/issues/4014. I just checked it with a repro from another related issue and it works fine. Please give it a try again.

**Comment by daveMueller** [comment]
I close this as the external issue is fixed. Feel free to reopen.
